pkg/row/copier_test.go

```diff
@@ -300,78 +325,11 @@ func TestCopierValidation(t *testing.T) {
 	assert.NoError(t, err)
 	defer db.Close()
 
-	t1 := table.NewTableInfo(db, "test", "t1")
-
-	// if the checksum is FALSE, the unique violation will cause an error.
-	_, err = NewCopier(db, t1, nil, NewCopierDefaultConfig())
+	// Test that NewCopier fails with nil chunker
+	_, err = NewCopier(db, nil, NewCopierDefaultConfig())
 	assert.Error(t, err)
 }
 
-func TestETA(t *testing.T) {
-	testutils.RunSQL(t, "DROP TABLE IF EXISTS testeta1, testeta2, _testeta1_new, _testeta2_new")
-	testutils.RunSQL(t, "CREATE TABLE testeta1 (a INT NOT NULL, b INT, c INT, PRIMARY KEY (a))")
-	testutils.RunSQL(t, "CREATE TABLE testeta2 (a INT NOT NULL auto_increment, b INT, c INT, PRIMARY KEY (a))")
-	testutils.RunSQL(t, "CREATE TABLE _testeta1_new (a INT NOT NULL, b INT, c INT, PRIMARY KEY (a))")
-	testutils.RunSQL(t, "CREATE TABLE _testeta2_new (a INT NOT NULL auto_increment, b INT, c INT, PRIMARY KEY (a))")
-	// high max value
-	testutils.RunSQL(t, "INSERT IGNORE INTO testeta2 VALUES (10000, 2, 3)")
```

Comment:
The copier is now no longer responsible for this. I've removed the test for now, but I may need to add more tests to replace it once I add the multi chunker.